src/tile-nina/tile-nina.scss

```diff
@@ -34,32 +24,57 @@
 
   .tile-nina-button {
     @include font($skytext-med);
-    margin: 10% auto 0;
+    margin: 2.5% auto 0;
     display: inline-block;
-    padding: .5em 1em;
-    border: 2px solid $sky-blue;
+    padding: .3em 1.3em;
+    border: 1px solid $white;
     border-radius: 4px;
-    color: $sky-blue;
-    font-size: 20px;
+    color: $white;
+    font-size: desktop-font(20);
+
+    @include tablet {
+      font-size: desktop-font(40);
+    }
+
+    @include mobile {
+      font-size: desktop-font(84);
+    }
   }
 
   &:hover .tile-nina-button {
-    color: $white;
-    background-color: $sky-blue;
+    background-color: $white;
+    color: $sky-blue;
   }
 
   &.small {
-    .tile-title,
-    .tile-details {
-      font-size: desktop-font(26);
+    .tile-title {
+      font-size: desktop-font(36);
       line-height: 1.2em;
+      color: $white;
+      padding-top: 1em;
```

Comment:
Being a bit picky but could we reduce this to 0.5em instead? There's a lot of padding at the top of the tile compared to the bottom:

![screen shot 2016-08-26 at 8 42 47 am](https://cloud.githubusercontent.com/assets/6919706/17997896/4c06df84-6b69-11e6-8ba2-46b7418c07a7.png)